.github/workflows/test_pypi_release.yml

```diff
@@ -0,0 +1,29 @@
+name: Test PyPI Release
```

Comment:
What is intended use-case for this action? Is it for testing if our github actions are working properly, or for us to make a test release  before doing the real one?

Reply:
I created it to test our GitHub action/workflow, but I guiess it might be better to test it on a specific branch to avoid merging it into the develop branch?